modules/linux_packages/manifests/python3_zstandard.pp

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class linux_packages::python3_zstandard {
+    require linux_packages::py3
+
+    # for the first run, this tries to run pip because pip3 didn't exist
+    # at the start at the start of the run. ugh. puppet bug in 5.5.1.
```

Comment:
Is the still a bug in puppet 6?  We should be using the latest puppet.

Reply:
It was always working in 6. Not sure why I thought we were locked at 5 (maybe the servers I looked at were using it or ???). It was awhile ago when I wrote that.

Will change.